streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestClient.java

```diff
@@ -211,7 +227,17 @@ public Double apply(Long value1, Long value2) {
                     "cntByCnt"
         ).to(stringSerde, longSerde, "tagg");
 
-        return new KafkaStreams(builder, props);
+        final KafkaStreams streamsClient = new KafkaStreams(builder, props);
```

Comment:
In this case I actually want the new change since I rely on exception handlers to catch problems.